src/styles/getTooltipStyles.js

```diff
@@ -56,22 +56,24 @@ function computeHorizontalShift(windowWidth, xOffset, componentWidth, tooltipWid
  *                           and the left edge of the wrapped component.
  * @param {Number} yOffset - The distance between the top edge of the window
  *                           and the top edge of the wrapped component.
+ * @param {Element} wrapper - The reference to the tooltip wrapper
  * @returns {Boolean}
  */
-function isOverlappingAtTop(xOffset, yOffset) {
+function isOverlappingAtTop(xOffset, yOffset, wrapper) {
```

Comment:
@parasharrajat I added a commit that prevents non-ideal UX when scrolling down while hovering an element with a tooltip.

Before:

https://github.com/Expensify/App/assets/2590174/804a1245-3bfc-4f03-a11c-0a8a2021dd84

After:

https://github.com/Expensify/App/assets/2590174/59963834-53f6-477c-b3da-4a27621bbada